specs/DesignDiscussions/Inconsistencies.md

```diff
@@ -0,0 +1,179 @@
+* Distinct implementations means that defaults can vary by platform (e.g. whether the `TextBlock` `wrap` property defaults to `true` if not specified)
+* Differing choices made by devs at implementation time can lead to divergent behaviors (e.g. whether an invalid property value is silently dropped vs. emitting a warning vs. failing to parse the card)
+* Supporting libraries and platform affordances available to us can ease or speed up implementation, but can also be the source of inconsistencies (e.g. differences in how a dev constructs .NET's `DateTime`, UWP's `DateTime`, C++'s `std::chrono::time_point`, and Javascript's `Date` -- instantiating each object with, for example, a string, can vary in permissivity)
+  * Indeed, we've had a few issues arise from differences between a hand-rolled solution and a pre-rolled solution (e.g. date/time handling (#1915) and support for markdown (#1984))
+
+### Rendering
+
+Rendering presents another avenue by which inconsistencies can sneak into Adaptive Cards. Some of these problems can be pretty tricky to solve.
+
+* Multiple implementations means that "gotcha" scenarios might not always be covered or may be covered inconsistently (e.g. what happens when you combine `maxLines`, markdown lists, and `wrap: true`?)
+* Platform-provided controls can vary widely in their behaviors and support. Even if divergent behavior is caught during implementation, it may be difficult or tricky to solve completely
+
+## Proposals
+
+* (P0) Change processes to prevent new inconsistencies from ever happening:
+  * Motivation: Inconsistencies are caught before they ship
+  * Tasks: 
+    * New bugs automatically filed with `inconsistency` tag, which is only removed once it's determined that the issue is *not* a consistency issue
+    * We can't ship if an `inconsistency`-tagged bug hasn't been resolved for this release
+    * Features that need implementation across all platforms are merged to a single branch and PR'd/tested for inconsistencies before being allowed to merge upstream
+      * Each feature is championed by a single dev or PM, who owns the feature from conception to first ship
+      * Champion creates a branch off of `master` (or whichever main branch is appropriate), naming it `feature/featureName` (let's call this the "feature branch")
+      * The champion is responsible for keeping the feature branch up to date with respect to `master` (or main branch)
+      * Workflow:
+        1. People working on the feature create a new branch off of the feature branch (let's call it the "working branch")
+        1. When feature work in a working branch is completed, it is PR'd to the feature branch
+        1. **The first PR to feature branch must include robust test cards that not only exercise the "golden path" for a feature, but every reasonable corner case. This PR must also include schema docs.**
```

Comment:
>cards [](start = 69, length = 5)

Discussion in the room: Call out where this card goes (\test) and that we also need a nice looking card for the feature to be shown on the webpage (in elements). Maybe a scenario card too.